muspinsim/muspinsim.xml

```diff
@@ -51,8 +51,8 @@
         <collection name="muspinsim_results" type="list" label="Muspinsim output collection">
             <discover_datasets pattern="__name__" directory="data" format="txt"/>
         </collection>
-        <data label="muspinsim log for $mu_sim.name" name="log_out" format="txt" from_work_dir="log_out.log"/>
-        <data label="fit report for $mu_sim.name" name="fit_report" format="txt" from_work_dir="fit_report.txt">
+        <data label="muspinsim log on ${on_string}" name="log_out" format="txt" from_work_dir="log_out.log"/>
+        <data label="fit report on ${on_string}" name="fit_report" format="txt" from_work_dir="fit_report.txt">
```

Comment:
Just `muspinsim.xml` missing the wrapper bump.

Reply:
fixed it.